src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php

```diff
@@ -90,10 +76,6 @@ public function process(ContainerBuilder $container)
             }
         }
 
-        if (!$container->has('http_kernel')) {
-            $container->removeDefinition('twig.controller.preview_error');
```

Comment:
This service was removed in 5.0 (ref: https://github.com/symfony/symfony/pull/33476/files#diff-af1c832537900d7a594b1a9249348b13L135)